newsroom/topics/views.py

```diff
@@ -176,49 +189,51 @@ def get_topic_url(topic):
     )
 
 
-@blueprint.route("/topic_share", methods=["POST"])
+@topic_endpoints.endpoint("/topic_share", methods=["POST"])
 @login_required
-async def share():
+async def share(args: RouteArguments, params: None, request: Request):
     current_user = get_user(required=True)
     data = await get_json_or_400()
     assert data.get("users")
     assert data.get("items")
     topic = get_entity_or_404(data.get("items")["_id"], "topics")
     for user_id in data["users"]:
-        user = get_resource_service("users").find_one(req=None, _id=user_id)
+        user_data = await UsersService().find_by_id(user_id)
+        user = user_data.dict(by_alias=True, exclude_unset=True)
```

Comment:
Use the new `to_dict` method we have, won't need to pass in `by_alias` or `exclude_unset` anymore then.